redbot/core/dev_commands.py

```diff
@@ -305,6 +309,22 @@ async def repl(self, ctx):
             except discord.HTTPException as e:
                 await ctx.send(_("Unexpected error: `{}`").format(e))
 
+    @repl.command()
+    async def pause(self, ctx, toggle: Optional[bool] = None):
```

Comment:
We shouldn't be using the behavior of d.py's `Optional` converter here. You can either remove the `Optional` or have the proper type hint using `NoParseOptional` from `redbot.core.commands` (`from redbot.core.commands import NoParseOptional as Optional`)